add documentation for Radius Dashboard (#1047)
radius-project/docs: docs/content/guides/tooling/dashboard/overview/index.md

---
type: docs
title: "Overview: Radius Dashboard"
linkTitle: "Overview"
description: "Learn about using the Radius Dashboard to visualize your application graph, environments, and recipes"
weight: 100
categories: "Overview"
tags: ["dashboard"]
---

> Radius Dashboard is the frontend experience for Radius and built on [Backstage](https://backstage.io/), an open-source platform for building developer portals that provides a rich set of components to accelerate UI development. The Radius Dashboard is a skinned deployment of Backstage that includes a set of plugins that provide the Radius experience. The components that make up the dashboard are built with extensibility in mind so that they can be used in other contexts beyond Backstage in the future.

Reviewer: Small nit: do we want to call this "Radius Dashboard", or "The Radius Dashboard"? The latter seems more natural to me, but overall I'm good with both

Reviewer: This first paragraph goes straight into implementation details which end-users may not necessarily care about. Instead, maybe the first paragraph should focus on why the Dashboard is useful to users, and then we have a dedicated section on implementation?

Author: I like it, updated to reflect your suggestion.

Key features of the Radius Dashboard currently include:

- _Application graph visualization_: A visualization of the [application graph]({{< ref "guides/author-apps/application/overview#query-and-understand-your-application-with-the-radius-application-graph" >}}) that shows how resources within an application are connected to each other and the underlying infrastructure.
- _Resource overview and details_: Detailed information about resources within Radius, including [applications]({{< ref "guides/author-apps/application/overview" >}}), [environments]({{< ref "guides/deploy-apps/environments/overview" >}}), and infrastructure.

> - _Recipes directory_: A listing of all the Radius [Recipes]({{< ref "guides/recipes/overview" >}}) available to the user for a given environment.

Reviewer: Instead of italics for the main points, I think bold text would look better here.

Author: good suggestion, updated

{{< alert title="Dashboard installation" color="warning" >}}

> The Radius Dashboard is installed by default as a part of your Radius initialization and deployment. To opt-out of installing the dashboard, you can use the `--TODO` flag when running `rad init`, `rad deploy`, or `rad run`. See more instructions in the [CLI reference documentation]({{< ref "/reference/cli" >}}).

Reviewer 2: ```suggestion
The Radius Dashboard is installed by default as a part of your Radius initialization and deployment. To opt-out of installing the dashboard, you can use the `--set dashboard.enabled=false` flag when running `rad install kubernetes`. See more instructions in the [CLI reference documentation]({{< ref "/reference/cli" >}}).
```

> The Radius Dashboard is installed by default as a part of your Radius initialization and deployment. To opt-out of installing the dashboard, you can use the `--set dashboard.enabled=false` flag when running `rad install kubernetes`. See more instructions in the [CLI reference documentation]({{< ref "/reference/cli" >}}).

Reviewer: Do we want to point to the CLI reference here? Would the installation docs make more sense?

Author: agreed, updated

{{< /alert >}}

> When you run your application with the `rad run` command, Radius creates a port-forward from `localhost` to port `TODO` inside the container that you may use to access your Radius Dashboard.

Reviewer 2: ```suggestion
When you run your application with the `rad run` command, Radius creates a port-forward from `localhost` to port `7007` inside the container that you may use to access your Radius Dashboard.
```

> When you run your application with the `rad run` command, Radius creates a port-forward from `localhost` to port `7007` inside the container that you may use to access your Radius Dashboard.

Reviewer: Can we add some headers to break up this page into sections? Looks like we have Overview, Key Features, Usage, and Reference documentation. Will help break up large chunks of text.

Author: good suggestion, updated

Alternatively, you can manually create a port-forward from `localhost` to the port number of your choice to provide access to your Radius Dashboard:

```bash

> kubectl port-forward svc/TODO 3000:80

Reviewer 2: kubectl port-forward --namespace=radius-system svc/dashboard 7007:80

> kubectl port-forward svc/TODO 3000:80

Reviewer 2: ```suggestion
kubectl port-forward --namespace=radius-system svc/dashboard 7007:80
```
